boot/modeenv.go

```diff
@@ -154,6 +155,10 @@ func modeenvFile(rootdir string) string {
 // ReadModeenv attempts to read the modeenv file at
 // <rootdir>/var/iib/snapd/modeenv.
 func ReadModeenv(rootdir string) (*Modeenv, error) {
+	if snapdenv.Preseeding() {
```

Comment:
what should be do about the reading/writing of modeenv done by the boot package itself? have you looked how much of it happens during preseeding? maybe none?

Reply:
As mentioned in the standup, I found SetNextBoot is the only trigger for this, and it's guarded now.